### DISCUSSION TITLE
TextEdit

### DISCUSSION THREAD

**Discussion by alexmyczko** [trunkmaster/nextspace#388] [Ideas]
Hi Sergii

Glad to see you back :) have you tried https://github.com/ericwa/TextEdit (which is an updated version than the one you use)? Screenshot: https://twitter.com/aieidotch/status/860148123108245504

**Comment by trunkmaster** [comment]
Hi Alex

Not yet. Thank you for information. Are there any improvements against version 4? I can't find Changelog or something.

**Comment by alexmyczko** [comment]
It is a more recent version. The differences I noticed are Page Layout is improved, as in custom paper size can be set.
sloccount on old TextEdit:
Total Physical Source Lines of Code (SLOC)                = 2,455
ericwa TextEdit:
Total Physical Source Lines of Code (SLOC)                = 2,869

however the original icon got broken, so needs to be taken over from the old version
https://twitter.com/aieidotch/status/1052662530332606473/photo/1

**Comment by onflapp** [comment]
Hi Alex,

I tried it and it looks it is very MacOS-oriented.
There are many features that do not appear to work under nextspace.

I would suggest to run with the current version for now and add things to it as needed.
